### PR TITLE
Add "Close Toilet Seat" section

### PR DESCRIPTION
**Pull request by matiasgarciaisaia**
Created by @nicolevasilevsky in #108, supplemented by #109, #110, and #111.

This pull request unifies those 4 in a single one.

It's still up for scientific review.

Co-authored-by: Nicole Vasilevsky <vasilevs@ohsu.edu>

### CODE REVIEW

**Comment by matiasgarciaisaia** [comment]
To be clear, I'm not advocating for the change, nor against it. I'm just unifying those into a single one to have the proper discussion in place.

**Comment by steesdale** [comment]
I am not aware of literature that shows fecal-oral as a relevant mode of transmission.  CDC states that "the risk is expected to be low based on data from previous outbreaks of related coronaviruses, such as severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS). There have been no reports of fecal-oral transmission of COVID-19 to date."  

https://www.cdc.gov/coronavirus/2019-ncov/php/water.html

To this end while its general good advice, I think it would detract from the more relevant advice like wash your hands, social distancing, etc... 

If more relevant research is available I would be happy to re-review...

**Comment by steesdale** [comment]
I suggest we close this unless new information comes to light.

File: _content/en/02-do/12-close-toilet-seat.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{:.only_toc}
+## Toliet seat
```
Comment (mverzilli) [inline _content/en/02-do/12-close-toilet-seat.md]: Typo here, would you fix?

File: _content/en/02-do/12-close-toilet-seat.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+{:.do.no_toc}
+## Close toliet seat before flushing
```
Comment (mverzilli) [inline _content/en/02-do/12-close-toilet-seat.md]: Same (toliet => toilet)

**Comment by mverzilli** [comment]
Based on @steesdale points and @nicolevasilevsky "thumbs up" reaction, I'm closing this.

**Pull request closed without merging**